crates/uv/tests/python_pin.rs

```diff
@@ -629,3 +630,39 @@ fn python_pin_resolve() {
         "###);
     });
 }
+
+#[test]
+fn python_pin_with_comments() -> Result<()> {
+    let context = TestContext::new_with_versions(&[]);
+
+    let child = context.temp_dir.child("foo");
+    child.create_dir_all()?;
+
+    let version_file = child.child(PYTHON_VERSION_FILENAME);
+    version_file.write_str("3.12\n\n# 3.11\n3.10\n\n")?;
```

Comment:
It'd be nice to use a `r#` string and split this over multiple lines for readability.